dodo.py

```diff
@@ -423,31 +421,33 @@ def task_dist():
 
 def task_dev():
     """setup up local packages for interactive development"""
-    if C.TESTING_IN_CI or C.DOCS_IN_CI or C.LINTING_IN_CI:
-        cwd = P.ROOT
-        file_dep = [
-            B.DIST / f"""{C.NAME.replace("-", "_")}-{D.PY_VERSION}-{C.NOARCH_WHL}"""
-        ]
-        args = [
-            *C.PYM,
-            "pip",
-            "install",
-            "-vv",
-            "--no-index",
-            "--find-links",
-            B.DIST,
-            C.NAME,
-        ]
-    else:
-        cwd = P.PY_SETUP_PY[C.NAME].parent
-        file_dep = [cwd / "src" / C.NAME / B.APP_PACK.name]
-        args = [*C.FLIT, "install", "--pth-file", "--deps=none"]
+    for py_name in [C.NAME, C.CORE_NAME]:
```

Comment:
doing this in a loop doesn't work very well for pip... perhaps generate a concrete `requirements.txt` that lists all of the things needed in a loop, and then have a single `pip` call that runs that?

Reply:
yeah I guess. Hoping we could just do `pip install -e .` at some point (#684) but that would be for later.

Reply:
If going with `pip`, with two inter-dependent packages it would still make sense to have a `requirements-editable.txt` with something like:

```
-e py_packages/jupyterlite_core
-e py_packages/jupyterlite
```

But then potentially turn on `--no-build-isolation` to avoid some of the "surprises" mentioned elsewhere.

Reply:
presumably covered by below punt